services/alerta/service.go

```diff
@@ -17,13 +17,19 @@ import (
 type Service struct {
 	configValue atomic.Value
 	logger      *log.Logger
+	client      *http.Client
 }
 
 func NewService(c Config, l *log.Logger) *Service {
 	s := &Service{
 		logger: l,
 	}
 	s.configValue.Store(c)
+	s.client = &http.Client{
```

Comment:
Here the use the `Store` method to store the new client. Much like the configValue line above.